src/core/dev/hw_queue_tx.cpp

```diff
@@ -2013,6 +2018,8 @@ bool hw_queue_tx::expand_doca_task_pool(bool is_lso)
         PRINT_DOCA_ERR(hwqtx_logerr, rc, "DOCA expand task pool (LSO=%d) failed", is_lso);
         return false;
     }
+
+    ++m_task_list_expand_count;
```

Comment:
1. Each expansion expands by "DOCA_EXPAND_BATCH_SIZE", not 1.
2. Please use 32K convention and not 32000, see other params for reference.

Reply:
Thanks @iftahl. very much appreciated :)
Fixed comments. let me know if it's OK by you.